monkestation/code/modules/ghost_players/arena/fight_button.dm

```diff
@@ -15,12 +15,23 @@
 	var/obj/item/weapon_of_choice = /obj/item/storage/toolbox
 	///the wager in monkecoins thats paid out to the winner
 	var/payout = 0
+	///list of weakrefs to spawned weapons for deletion on duel end
+	var/list/spawned_weapons = list()
 
+	///what weapons can players choose to duel with
 	var/list/weapon_choices = list(
 		/obj/item/storage/toolbox,
 		/obj/item/knife/shiv,
 		/obj/item/grenade/clusterbuster,
 		/obj/item/spear/bamboospear,
+		/obj/item/reagent_containers/spray/chemsprayer/magical, //unsure if this would cause issues but they do already have access to a full chem lab so it should be fine
+		/obj/item/gun/energy/laser/instakill, //first to hit the other wins, very fast matches
+		/obj/item/melee/baton/security/loaded,
+		/obj/item/chainsaw,
+		/obj/item/melee/energy/sword/saber,
+		/obj/item/book/granter/martial/cqc/fast_read,
+		/obj/item/gravity_gun,
```

Comment:
this one is a no go because it can be used to grab people out of the arenas, specifically the fight button

Reply:
ah yeah, ok